### PR TITLE
fix: [web] svg class selector

### PR DESCRIPTION
**Pull request by OrenZak**

### CODE REVIEW

File: webDemo/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "@react-native-community/netinfo": "^9.3.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-native": "npm:react-native-web@0.19.4",
```
Comment (ethanshar) [inline webDemo/package.json:22]: What's that?
Why do we need this?

File: src/components/button/index.tsx [hunk 2]
```diff
@@ -301,7 +301,13 @@ class Button extends PureComponent<Props, ButtonState> {
       } else {
         if (Constants.isWeb) {
           return (
-            <Icon style={iconStyle} tintColor={Colors.$iconDefault} source={iconSource} testID={`${testID}.icon`}/>
+            <Icon 
+              id={`button_icon_${this.props.id}`}
```
Comment (ethanshar) [inline src/components/button/index.tsx:305]: Where the `id` prop is coming from? 
have you declared in Button props? or in Icon component's props?

File: src/components/svgImage/index.web.tsx [hunk 3]
```diff
@@ -8,27 +8,37 @@ export interface SvgImageProps {
...
-    const {postcss, cssjs} = PostCssPackage;
-    postcss()
-      .process(styleObj, {parser: cssjs})
-      .then((style: {css: any}) => {
-        const svgPathCss = (styleObj?.tintColor) ? `svg path {fill: ${styleObj?.tintColor}}` : '';
-        setSvgStyleCss(`svg {${style.css}} ${svgPathCss}}`);
-      });
-  };
-
+  const className = `${props?.id ?? 'svg-icon'} `;
```
Comment (ethanshar) [inline src/components/svgImage/index.web.tsx:26]: maybe generate an uniq random id so each classname will have a different scope

**Comment by mendyEdri** [comment]
Let's prioritize a complete fix for this issue, with this PR this is the result:

image

**Comment by ethanshar** [comment]
@mendyEdri @OrenZak 
Can we close this one, I think it's not relevant anymore

**Comment by mendyEdri** [comment]
> @mendyEdri @OrenZak Can we close this one, I think it's not relevant anymore

from my side yes, we will switch to prod

**Comment by OrenZak** [comment]
> @mendyEdri @OrenZak Can we close this one, I think it's not relevant anymore

why does its not relevant ?
we need to have a unique id for a iconButton of web, and apply the styling via css selector.
is it already on master ?

**Comment by mendyEdri** [comment]
> > @mendyEdri @OrenZak Can we close this one, I think it's not relevant anymore
> 
> why its not relevant ? we need to have a unique id for a iconButton of web, and apply the styling via css selector. is it already on master ?
> 
> @AmitShwarts , have you done something related to that ? we should solve the case where we have multi icon buttons.

We thought [this PR ](https://github.com/wix/react-native-ui-lib/pull/2728) supposed to replace yours

**Comment by AmitShwarts** [comment]
> 

Hey guys, I didn't know about this PR and we merged yesterday a different PR that solves this and some more bugs in web svg image.

**Pull request closed without merging**